src/event_emitter.rs

```diff
@@ -1,10 +0,0 @@
-use tree_sitter_grep::{tree_sitter::Node, SupportedLanguage};
```

Comment:
Got rid of event emitter and switch eslint-builtin code path analysis to use a provided type instead